components/dropdown/dropdown-content-styles.js

```diff
@@ -19,21 +21,24 @@ export const dropdownContentStyles = css`
 	}
 
 	:host([theme="dark"]) {
+		--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation-dark;
+		--d2l-dropdown-animation-name: d2l-dropdown-animation-dark;
 		--d2l-dropdown-background-color: #292a2b; /* ferrite @ 50% */
 		--d2l-dropdown-border-color: var(--d2l-color-ferrite);
 		--d2l-dropdown-foreground-color: var(--d2l-color-mica);
 		--d2l-dropdown-shadow-color: rgba(0, 0, 0, 1);
+		opacity: 0.9;
```

Comment:
Are we sure we want to use `opacity` for this?  While it is convenient that is also makes the menu semi-transparent (which currently has a solid background color applied), it also makes the text, and everything else in the dropdown semi-transparent.

Reply:
Yeah I know. I tried really hard to just apply an `rgba()` with alpha to the existing menu background colours -- the issue is that all 3 of the dropdown container, the `<d2l-menu>` container AND the individual `<d2l-menu-item>`s have a background colour set (to white in non-dark mode, different on-hover for the items). I can't have all 3 be 90% opaque... otherwise you end up with much less opacity than intended -- and different opacity depending on whether a menu is contained in a dropdown or not.

To get around that, we'd need to remove the background colour from dropdowns when a menu is contained inside (which it doesn't really know) and the menu container as well, which has implications for custom menu items and the header/footer areas. This solution was easier. :)

Reply:
I see - all makes sense.  It seems like even if the nested component tries to set `opacity: 1`, it doesn't really seem to override the ancestor's opacity... it just means `opacity: 1` in the context of the `opacity: 0.9`.  The transparency we are using is pretty subtle, so maybe it's ok?... but if a higher transparency were used we might not be able to use this approach.

Reply:
Yeah it's very subtle. The other drawback to this approach is that it's only being applied to dropdown menus -- menus on their own (which are rare) don't get it. Fine for media player, but if we were doing a larger "dark mode" thing it may not be.

Reply:
Good point.  The only other spot I know menu is used outside of dropdowns is the mobile fly-out.  Not sure if we'd want transparency applied there in such a case or not.  🤷‍♂️